src/bin/vip-app-deploy.ts

```diff
@@ -108,29 +80,31 @@ export async function promptToContinue( params: PromptToContinueParams ) {
 }
 
 export async function appDeployCmd( arg: string[] = [], opts: Record< string, unknown > = {} ) {
-	const app = opts.app as App;
-	const env = opts.env as AppEnvironment;
+	const app = opts.app as string | number;
+	const env = opts.env as string | number;
 
 	const [ fileName ] = arg;
 	const fileMeta = await getFileMeta( fileName );
-	const inputBasename = fileMeta.basename;
+	const fileInputBasename = fileMeta.basename;
 
 	debug( 'Options: ', opts );
 	debug( 'Args: ', arg );
 
-	const appId = env.appId as number;
-	const envId = env.id as number;
+	debug( 'Validating custom deploy key...' );
+	const { appId, envId, ...validatedArgs } = await validateCustomDeployKey( app, env );
+	console.log( { appId, envId, validatedArgs } );
```

Comment:
Let's just do `debug()` for this info, I don't think the user needs this info. Can also remove too.

Reply:
Will remove. Debugging while I test out all the various ways an app/env can be input 🙃